docs/usage/misc/persistence.md

```diff
@@ -50,3 +50,17 @@ persistence(:influxdb) do
   Item1.average_since(12.hours)
 end
 ```
+
+## Quantity
+
+Most persistence functions (`xxx_since` and `evolution_rate`) return a DecimalType object. This can be converted to Quantity using the `|` operator for comparison against other Quantity based data.
+
+### Example
+
+```ruby
+# Item1's persistence data is stored in Watt unit
```

Comment:
> Does this all still work when we wrap it in the units blocks?
> 
> ```
> units('W') do
>   if Item1.average(since(1.hour) > '2 kW'
> end
> ```

This won't work in the current state of the PR, because Item1.average_since(1.hour) is still a DecimalType at this point. It needs to be converted to Quantity first by the `|` operator.

If my proposal in the comment above is implemented, then the units block isn't even needed, because it will derive the (base) unit from the number item. By `base` unit, I mean `W` as opposed to `kW`, for example.

Reply:
So for this to work, we would need to monkeypatch decimal type in the same way we handle NumberItem, correct?

Not proposing we do it in this PR, but I will open a separate issue on it if my assertion is correct.

It would still be needed for items that lack a quantity.

Reply:
> So for this to work, we would need to monkeypatch decimal type in the same way we handle NumberItem, correct?

We are altering the decimaltype monkey patch to add the `|` operator in this PR